bibformat/format_elements/bfe_INSPIRE_links.py

```diff
@@ -71,8 +71,14 @@ def format_element(bfo, default='', separator='; ', style='',
 
         if provenance == 'KEKSCAN':
             extid = extid.replace("-", "")
-            links.append('<a%s href="%s%s"> KEK scanned document</a>' %
-                         (style, KEK, extid))
+            if not extid.startswith('20'):
+                year = '19' + extid[:2]
+            else:
+                year = extid[:4]
+                extid = extid[2:]
+            prefix = exitd[:4]
```

Comment:
another snafu: "prefix" has some other meaning in bibformat and leads to a prefix being printed in front of the format element. Doh!